scripts/globals/items/pinch_of_mana_powder.lua

```diff
@@ -0,0 +1,25 @@
+-----------------------------------------
+-- ID: 4255
+-- Item: Pinch of Mana Powder
+-- Item Effect: Restores 25% MP AoE 10' radius
+-----------------------------------------
+require("scripts/globals/status")
+require("scripts/globals/msg")
+-----------------------------------------
+-- OnItemCheck
+-----------------------------------------
+
+function onItemCheck(target)
+	return 0
```

Comment:
you still have tabs sneaking in, fix your editor

Reply:
I blame my fat fingers

Reply:
your editor should be placing 4 spaces when you press tab. NOBODY manually taps out 4 spaces man. every editor worth using has this feature you just need to set it, or use the editorconfig plugin for it.

Reply:
copy pasta ninja tabs should be gone now